### PR TITLE
Replace revenue with revenue_url call (resolve #115)

### LINKED ISSUE #115: Upgrade to new revenue endpoint
Instead of calculating revenue for each instructor response, we want to do that async with a url. The revenue block was particularly problematic for admins as EVERY instructor was calculated.

see: https://github.com/eggheadio/egghead-rails/commit/ea257c9bc73ef1e8664fe7b1954403ce18a4ef73

We need to migrate to `revenue_url` so that the `revenue` block can be removed.

### PR DESCRIPTION
**Pull request by trevordmiller**
# Changes

- Replace `instructor.revenue` with `instructor.revenue_url` call
- Upgrade to latest `egghead-ui` to get padding around `Loading`
- Fix `egghead.dev` base url
- Add log in info in README

# Result For User

No change other than the wording "Previous" is now "Last" to be more clear that the last  includes the current estimated earnings as well.

---

### CODE REVIEW

File: src/screens/Overview/index.js [hunk 1]
```diff
@@ -22,7 +23,14 @@ export default ({instructor}) => (
...
         <div className='mb4'>
-          <Stats instructor={instructor} />
+          <Request url={instructor.revenue_url}>
+            {({data}) => (
+              <Stats 
+                instructor={instructor}
+                revenue={data.revenue}
+              />
+            )}
+          </Request>
```
Comment (trevordmiller) [inline src/screens/Overview/index.js]: Ahhhh yeeeah. Love how simple it is to add data from an endpoint now.

File: src/screens/Overview/index.js [hunk 2]
```diff
@@ -22,7 +23,14 @@ export default ({instructor}) => (
     aside={
       <div>
         <div className='mb4'>
-          <Stats instructor={instructor} />
+          <Request url={instructor.revenue_url}>
+            {({data}) => (
+              <Stats 
+                instructor={instructor}
+                revenue={data.revenue}
```
Comment (joelhooks) [inline src/screens/Overview/index.js]: I pushed the change removing `revenue` prop so you can just pass in data.

**Review by joelhooks — CHANGES_REQUESTED** [review]

File: src/screens/Overview/index.js (hunk 2 above)
Comment (trevordmiller) [reply src/screens/Overview/index.js]: Oh gotcha, I misunderstood - thought you meant the `instructor.revenue` prop. I'll update this. Thanks.

File: src/screens/Overview/index.js [hunk 3]
```diff
@@ -22,7 +23,14 @@ export default ({instructor}) => (
     aside={
       <div>
         <div className='mb4'>
-          <Stats instructor={instructor} />
+          <Request url={instructor.revenue_url}>
+            {({data}) => (
+              <Stats 
+                instructor={instructor}
+                revenue={data}
```
Comment (trevordmiller) [inline src/screens/Overview/index.js:30]: removed `data.revenue` now that `revenue` is the data directly

**Review by joelhooks — APPROVED** [review]

**Pull request merged**